docs/getting_started.md

```diff
@@ -0,0 +1,325 @@
+# Getting started
+
+## Data generation process
+
+The data generation process uses this analogy: generated data flows from source to sink.
+
+To generate data is then necessary to define:
+
+* `source`: what data is generated, eg. data model
+* `sink`: where data is sent to, eg. ES index
+* `flow`: how data is transmitted, eg. how fast or how much?
+* `schema`: fields definition, eg. ECS 8.2.0
+
+Each of the above is handled by its own REST API endpoint. An arbitray
+number of sources, sinks, flows and schemas can be defined on the same
+server.
+
+## Install
+
+Currently Geneve is packaged only for [Homebrew](https://brew.sh), you
+need first to install the Geneve tap
+
+```shell
+$ brew tap elastic/geneve
+```
+
+then the tool itself
+
+```shell
+$ brew install geneve
+```
+
+## REST API server
+
+Data is generated by the Geneve server, you start it with
+
+```shell
+$ geneve serve
+2023/01/31 16:40:23 Control: http://localhost:9256
+```
+
+The server keeps the terminal busy with its logs, to stop just press `^C`.
+The first line in the log shows where to reach it, this is the base url of
+the server, all the API endpoints are reachable (but not browseable) under
+`api/`.
+
+For the rest of this document we'll assume that the following shell
+variables are set:
+
+* `$GENEVE` points to the Geneve server, url `http://localhost:9256`
+* `$TARGET_ES` is the url of the target Elasticsearch instance
+* `$TARGET_KIBANA` is the corresponding Kibana's url
```

Comment:
nit: Would be helpful UX to have defaults here, e.g if not set fallback to `http://localhost:9200` for ES, and `http://localhost:5601` for Kibana.

Reply:
I second your point. I think I could add a built-in `devel` (or `local` or `default`) sink in the REST API server that points to these well known instances. I'm less capable to nominate such fall back at the index level, maybe a plain `geneve-test` (or `geneve-devel` or `geneve-<date>`) index?

Reply:
IMHO since this is a documentation PR we should just focus on the documentation.  Actual feature changes should be done in another PR.

Reply:
> IMHO since this is a documentation PR we should just focus on the documentation. Actual feature changes should be done in another PR.

++, sorry for sneaking feature requests in this review. It was my first time going through setting things up and working with the library, so I just wanted to be sure to capture these little enhancements along the way. Will open any more detailed requests as dedicated issues in this repo 👍

Reply:
That's ok, I'm extremely interested in your suggestions. Feel free to open issues as needed. Thanks for your feedback.